macosx/kext/AudioEngine.cpp

```diff
@@ -64,7 +64,8 @@ bool kXAudioEngine::init(kx_hw *hw_)
 		// Use 4 times the buffer size to prevent some sort of underrun on
 		// Mavericks (related to Timer Coalescing?) causing playback crackle
 		// until streams are restart (e.g. stop/start playback)
-    n_frames = (int)(4 * (hw->mtr_buffer.size * 8 / bps / n_channels));
+//    n_frames = (int)(4 * (hw->mtr_buffer.size * 8 / bps / n_channels));
+     n_frames = (int)(hw->mtr_buffer.size * 8 / bps / n_channels);
```

Comment:
The comment above suggests that on Mavericks there's a buffer underrun. I wonder if it is specific to some earlier macOS release or to all future releases as well. Not sure how removing *4 multiplier affects other systems.

Reply:
When size*4, in recorded sound have gaps.